server/sonar-server/src/main/java/org/sonar/server/computation/component/ProjectSettingsRepository.java

```diff
@@ -26,7 +26,7 @@
 /**
  * Lazy loading of project settings.
  *
- * Should be refactored to be able to load settings from any components.
+ * TODO Should be refactored to be able to load settings from any components.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)